core/server/api/v2/slugs.js

```diff
@@ -0,0 +1,22 @@
+const models = require('../../models');
+
+module.exports = {
+    docName: 'slugs',
+    generate: {
+        options: [
+            'include',
+            'type',
+            'name'
+        ],
+        permissions: true,
+        query(frame) {
+            const allowedTypes = {
+                post: models.Post,
+                tag: models.Tag,
+                user: models.User,
+                app: models.App
+            };
+            return models.Base.Model.generateSlug(allowedTypes[frame.options.type], frame.options.data.name, {status: 'all'});
```

Comment:
`allowedTypes[frame.options.type]` - Should something like this go into input serializer or this is right place to do it? Was not completely sure about this case